docs: Update development.md

richb-hanover opened the pull request:
Second attempt to create a PR - this is patch-3

I don't know what happened to the previous PR - It never appeared in the PRQL repo, nor could I push these changes (using patch-2)

richb-hanover committed:
Update development.md

Second attempt to create a PR

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

max-sixty commented on web/book/src/project/contributing/development.md:
```diff
@@ -435,37 +437,40 @@ task run-playground
-   [Changelog](https://github.com/PRQL/prql/blob/main/CHANGELOG.md). GitHub will
-   produce a draft version at <https://github.com/PRQL/prql/releases/new>,
-   including "New Contributors".
+- [ ] PR & merge an updated
```
Do you know whether these render reasonably on the mdbook?

max-sixty commented on web/book/src/project/contributing/development.md:
```diff
@@ -435,37 +437,40 @@ task run-playground
-   commit.
-6. Check whether there are [milestones](https://github.com/PRQL/prql/milestones)
-   that need to be pushed out.
+- [ ] Review the **Current Status** on the README.md to ensure it reflects the
```
Maybe at the end? It doesn't need to block the other things but I agree it's a good time to update it (your last update was great!)

richb-hanover replied:
OK. Done. But see #3778

max-sixty commented on web/book/src/project/contributing/development.md:
```diff
@@ -88,12 +88,14 @@ since it relies on `brew`.
 and a
 [pre-built dev container base Docker image](https://github.com/PRQL/prql/pkgs/container/prql-devcontainer-base).
+Dev Containers are specified at
+[https://containers.dev/](https://containers.dev/)
```
Link is great but maybe we don't need this now you added it to the title?

richb-hanover replied:
Actually, I took the link _out_ of the title/heading because there was no link icon when I viewed the Markdown. (Those link icons were present for Options 1, 2, and 4)

But when looking at the Book directly either one looks good. Let's leave this like this.

richb-hanover committed:
Merge branch 'main' into patch-3

richb-hanover committed:
Editorial tweaks:
- Remove abortive attempt at `- [ ] ...` checkboxes (They force a newline after the checkbox)
- Move "Check Current Status" to end of the list

max-sixty commented on web/book/src/project/contributing/development.md:
```diff
@@ -435,37 +437,44 @@ task run-playground
-   [Changelog](https://github.com/PRQL/prql/blob/main/CHANGELOG.md). GitHub will
-   produce a draft version at <https://github.com/PRQL/prql/releases/new>,
-   including "New Contributors".
+- PR & merge an updated
```
Why the change from numbered? I obv don't have a super-strong view, but I had thought that making them numbered showed they were ordered?

richb-hanover replied:
Good point. Updated.

richb-hanover committed:
Switch to numbered points in the Releasing section

richb-hanover committed:
Merge branch 'main' into patch-3

max-sixty commented on web/book/src/project/contributing/development.md:
```diff
@@ -451,21 +453,28 @@ Currently we release in a semi-automated way:
-    interprets linebreaks as newlines. I haven't found a better way of editing
-    the markdown to look reasonable than manually editing the text.}}, enter the
-   tag to be created, and hit "Publish".
+       maintainers have access to this page.}}, copy the changelog entry into the
+   release description{{footnote: Unfortunately GitHub's markdown parser
+        interprets linebreaks as newlines. I haven't found a better way of editing
+        the markdown to look reasonable than manually editing the text.}}, enter
+   the tag to be created, and hit "Publish".
```
Some odd indents here...

richb-hanover replied:
I'm pretty sure this PR addresses the odd indents that are currently in the Book (https://prql-lang.org/book/project/contributing/development.html)

The problem is that the `admonish note` cannot be indented: it gets screwed up when there are spaces at the front. But maybe I'm not seeing what you noticed

max-sixty replied:
OK cool, it does look good in HTML!
